### PR TITLE
Add support for swapping TradeHistory ticker symbols due to coin name change

### PR DESCRIPTION
**Pull request by anson-vandoren**
A fix for issue #9 (docstrings included this time :) )

Added a new example if you think that is helpful

### CODE REVIEW

File: ccgains/trades.py [hunk 1]
```diff
@@ -475,6 +475,43 @@ def add_missing_transaction_fees(self, raise_on_error=True):
...
+        if not isinstance(changes, dict):
+            log.warning('`update_ticker_names` expected a dict, but got %s' % type(changes))
+            return
+        count = {}
+        for i in range(len(self.tlist)):
+            for old, new in changes.items():
+                if self.tlist[i].buycur == old:
+                    self.tlist[i].buycur = new
+                elif self.tlist[i].sellcur == old:
+                    self.tlist[i].sellcur = new
```
Comment (probstj) [inline ccgains/trades.py]: Could you also change `self.tlist[i].feecur` here?

**Comment by probstj** [comment]
Great, a very good solution :)

**Comment by anson-vandoren** [comment]
Thanks for catching that! Done and should be ready to merge now.

**Comment by probstj** [comment]
Merged. Thank you! :)

**Pull request merged**